coveo/ui-kit
# fix(headless): when a tab is selected, the pagination resets to 1

[SFINT-4860](https://coveord.atlassian.net/browse/SFINT-4860)

Small change so when we change tabs, it will reset to the first page

Steps to test:
1. Open a atomic(or quantic) insight panel with multiple tabs and pages
2. go to page 2 on the first tab
3. switch to another tab
4. Page should be changed to 1 after swaping tabs

[SFINT-4860]: https://coveord.atlassian.net/browse/SFINT-4860?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
